app/controllers/api/v1/admin/server_recordings_controller.rb

```diff
@@ -4,6 +4,10 @@ module Api
   module V1
     module Admin
       class ServerRecordingsController < ApiController
+        before_action only: %i[index resync] do
```

Comment:
Conceptually, admin controllers if found should concern one area of GL3 admin panel and `ensure_authorized(...)` for all actions by default,